pkg/auth/routes.go

```diff
@@ -32,23 +32,31 @@ func NewRouter(p *deps.RootProvider, configSource *configsource.ConfigSource, st
 		p.Middleware(newPanicLogMiddleware),
 		p.Middleware(newSessionMiddleware),
 		p.Middleware(newSecHeadersMiddleware),
-		httproute.MiddlewareFunc(httputil.NoCache),
+	)
+
+	staticChain := httproute.Chain(
+		rootChain,
+		p.Middleware(newCORSMiddleware),
+		httproute.MiddlewareFunc(httputil.ETag),
```

Comment:
Seems better to separate chain for static and oauth config? Although we can use same middleware now, we may want to have different caching strategy in future. The name `staticChain` makes me think we only use it in static asset.